Src/.editorconfig

```diff
@@ -1,6 +1,15 @@
 # Remove the line below if you want to inherit .editorconfig settings from higher directories
 root = true
 
+# Ignore paths
+[/sarif-pattern-matcher/Src/sarif-sdk/src/sarif/Autogenerated/**]
```

Comment:
>matcher [](http://example.com/codeflow?start=16&length=7)

I'm having trouble making sure this folder isn't analyzed in the IDE. Do you know whether this should be working?

Reply:
what are you trying to trigger here?
for this to work we would need to change the sarif-sdk and place a file like this in the autogenerated folder.

Reply:
The IDE is showing analysis errors for the Autogenerated folder.